rest-server/src/components/auth/auth.test.js

```diff
@@ -0,0 +1,44 @@
+import request from 'supertest';
+import { dropTables, syncTables } from "../../lib/sqlScripts";
```

Comment:
Let's put these on different lines. It helps us see what was changed more easily.

```javascript
import {
  dropTables,
  syncTables
} from '../../lib/sqlScripts'
```

Also, single quotes!!